recipe/meta.yaml

```diff
@@ -1,171 +1,158 @@
-  sha256: a4f154fa98cde5974117e05af444e0bed84cf431b8129c78a2de03a1ff7201c9  # [win and py==37]
+  url: https://github.com/tensorflow/tensorflow/archive/v{{ version }}.tar.gz
+  sha256: 26c833b7e1873936379e810a39d14700281125257ddda8cd822c89111db6f6ae
+
   patches:
-    - patches/0001-add-rt-to-linkopts.patch                                     # [linux]
-    - patches/0002-stable-image-rotate-test.patch                               # [linux]
-    - patches/0003-relax-tolerence-in-special_math_test.patch                   # [linux]
-    - patches/0004-skip-read-only-debugger_cli_common-test.patch                # [linux]
-    #- patches/0005-fix-lookup_ops-test.patch                                    # [linux]
-    - patches/0006-relax-precision-in-matrix_log_op_test.patch                  # [linux]
-    #- patches/0007-Add-init-symbol-names-prefixed-with-an-underscore.patch      # [osx]
-    #- patches/0008-Add-alternate-clock_gettime-implementation-for-macOS.patch   # [osx]
-    - patches/unbundle_llvm.patch                                               # [linux]
+    - patches/0001-relax-tolerence-in-special_math_test.patch
+    - patches/0002-skip-read-only-debugger_cli_common-test.patch
+    - patches/0003-relax-precision-in-matrix_log_op_test.patch
+    - patches/0004-Unbundle-LLVM.patch
+    - patches/0005-rerun-generate_llvm_build.py.patch
 
 build:
   number: 0
   skip: True  # [not x86_64]
-  skip: True  # [win and vc<14]
 
 requirements:
   build:
-    - {{ compiler('c') }}       # [linux]
-    - {{ compiler('cxx') }}     # [linux]
-    - bazel >=0.24.1,<=0.25.2   # [linux]
-    - nasm                      # [linux]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    # see https://github.com/tensorflow/tensorflow/blob/v{{ version }}/.bazelversion
+    - bazel ==3.1.0
+    - nasm
   host:
     # conda build requirements
     - python
     - pip
-    - setuptools <=39.1.0
-    - swig                      # [linux]
-    # requirements specified by the package itself
-    - absl-py >=0.7.0
-    - astor >=0.6.0
-    - gast >=0.2.0
-    - google-pasta >=0.1.6
-    - keras-applications >=1.0.6
-    - keras-preprocessing >=1.0.5
-    - llvmdev >=9.0.0,<10.0.0    # [linux]
-    - zlib                      # [linux]
-    - libpng                    # [linux]
-    - libcurl                   # [linux]
-    - nsync                     # [linux]
-    - numpy 1.16.1
-    - six >=1.10.0
-    - protobuf >=3.6.1
+    # TODO: potentially remove these?
+    - llvmdev >=9.0.0
+    - zlib
+    - libpng
+    - libcurl
+    - curl         # [win]
+    - unzip        # [not win]
+    - zip          # [not win]
+    - m2-unzip     # [win]
+    - m2-zip       # [win]
+    - openjdk >=8
+    - nsync
+    # requirements specified by the package itself, see
+    # github.com/tensorflow/tensorflow/blob/v{{ version }}/tensorflow/tools/pip_package/setup.py
+    - absl-py >=0.10.0
+    - astunparse >=1.6.3
+    - flatbuffers >=1.12.0
```

Comment:
The correct one is `python-flatbuffers`. However, conda-forge/python-flatbuffers-feedstock#2 hasn't been merged yet.